json-endpoints/jetpack/class.jetpack-json-api-get-comment-backup-endpoint.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+class Jetpack_JSON_API_Get_Comment_Backup_Endpoint extends Jetpack_JSON_API_Endpoint {
+	// /sites/%s/comments/%d/backup      -> $blog_id, $comment_id
+
+	protected $needed_capabilities = array();
+	protected $comment_id;
+
+	function validate_input( $comment_id ) {
+		if ( empty( $comment_id ) || ! is_numeric( $comment_id ) ) {
+			return new WP_Error( 'comment_id_not_specified', __( 'You must specify a Comment ID', 'jetpack' ) );
```

Comment:
it would be good to specify the server error code here as appropriate in this case 400.